languages/ruby/reference/exercise-concepts/nucleotide-count.md

```diff
@@ -0,0 +1,44 @@
+# Concepts of nucleotide-count
+
+## General
+
+- class methods: used as the starting point of the exercise. They can be named with `self.[method_name]` notation or with a `self` block.
+- module methods: depending on the approach, they can be used instead of the class methods.
+- factory pattern and object creation: the method `from_dna` can be used to create instances of the main class.
+- instance variables: @variables are needed to store the common value/state for all object methods.
+- accessors: `att_reader` can be used to access instance variables.
+- access modifiers: `private` can be used to restrict access to the readers.
+- method definition: the `histogram` method needs to be defined.
+- method arguments: the method `from_dna` needs an argument.
+- string: one string is passed as the input.
+- chars: it's necessary to convert the string into an array of chars.
```

Comment:
It is not necessary, but one way to do so.

Reply:
I would use the full word when not using the method name... "characters" when speaking of characters, and `chars` when referring to the method of that name.